fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/BlockEntityRendererRegistry.java

```diff
@@ -28,7 +28,10 @@
 
 /**
  * Helper class for registering BlockEntityRenderers.
+ *
+ * <p>Use {@link net.minecraft.client.render.block.entity.BlockEntityRendererFactories#register(BlockEntityType, BlockEntityRendererFactory)} instead.
```

Comment:
This message could be made more consistent with other transitive access widener deprecations:

https://github.com/FabricMC/fabric/blob/5eb68ef24d677c538ad2f6caba2832366835bb33/deprecated/fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/FabricLootSupplier.java#L30